src/bindings/python/requirements_test.txt

```diff
@@ -24,12 +24,14 @@ flake8_commas
 flake8_pep3101
 flake8_quotes
 mypy
-onnx==1.12.0
+onnx==1.12.0; platform_system == "Linux" and platform_machine == armv7l or platform_system == "Linux" and platform_machine == aarch64
```

Comment:
why do we need it? I suppose this version is available for each OS

Reply:
condition here is wrong.